CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ option(CLANG_TIDY "Perform linting with clang-tidy" OFF)
 option(SANITIZERS "Enable sanitizers" OFF)
 option(MLS_NAMESPACE_SUFFIX "Namespace Suffix for CXX and CMake Export")
 
+set(BORINGSSL_ROOT_DIR "" CACHE STRING "Root of BoringSSL directory (non-empty string forces use of BoringSSL)")
```

Comment:
This isn't required, but I think it'll highlight the option in the GUI with a helpful description.